### PR TITLE
docs: add AVM notice

### PR DESCRIPTION
**Pull request by matt-FFFFFF**
@lonegunmanb as discussed

/cc @jtracey93 @PmeshramPM

### CODE REVIEW

**Review by lonegunmanb — APPROVED** [review]
LGTM! Thanks @matt-FFFFFF but I'd like to get approval from @grayzu .

**Comment by lonegunmanb** [comment]
I'll merge this pr if @grayzu haven't request change tomorrow.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Azure Terraform Verified Modules
 
+## ⚠️ Notice
+
+This program is merging with [Azure Verified Modules](https://aka.ms/avm) to provide a consistent user experiance across IaC languages on Azure.
```
Comment (zioproto) [inline README.md:5]: typo: experiance -> experience

File: README.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+***Existing TFVM modules will continue to be supported***, but for new projects you are advised to check the AVM [available modules](https://azure.github.io/Azure-Verified-Modules/indexes/terraform/) and use those if available.
+New modules will be created under the branding of AVM, so please check the website for more inforamtion.
```
Comment (zioproto) [inline README.md:7]: typo: inforamtion -> information

**Review by zioproto — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (lonegunmanb) [reply README.md:7]: Good catch, thanks @zioproto!

**Review by lonegunmanb — CHANGES_REQUESTED** [review]
Hi @matt-FFFFFF would you please check the issues found by @zioproto ? Thanks!

**Comment by matt-FFFFFF** [comment]
Fixed spelling - ready for re-review

**Review by lonegunmanb — APPROVED** [review]
Thanks @matt-FFFFFF , LGTM!

**Pull request merged**